BitwardenShared/UI/Platform/Application/Views/ToastView.swift

```diff
@@ -27,28 +49,42 @@ struct ToastView: View {
 
     var body: some View {
         if let toast {
-            Text(toast.text)
-                .styleGuide(.subheadline, weight: .semibold)
-                .multilineTextAlignment(.center)
-                .dynamicTypeSize(...DynamicTypeSize.accessibility2)
-                .id(toast.id)
-                .padding(14)
-                .foregroundColor(Asset.Colors.textReversed.swiftUIColor)
-                .frame(minWidth: 300, minHeight: 46)
-                .background(Asset.Colors.backgroundAlert.swiftUIColor)
-                .clipShape(RoundedRectangle(cornerRadius: 24))
-                .accessibilityElement(children: .combine)
-                .padding(.horizontal, 16)
-                .task(id: toast.id) {
-                    do {
-                        try await Task.sleep(nanoseconds: 3 * NSEC_PER_SEC)
-                        withAnimation {
-                            self.toast = nil
-                        }
-                    } catch {
-                        // No-op: Skip the animation if the task/sleep is cancelled.
+            VStack(alignment: .leading, spacing: 4) {
+                Text(toast.title)
+                    .styleGuide(
+                        .headline,
+                        weight: .bold,
+                        includeLinePadding: false,
+                        includeLineSpacing: false
+                    )
+
+                if let subtitle = toast.subtitle {
+                    Text(subtitle)
+                        .styleGuide(.callout)
+                }
+            }
+            .dynamicTypeSize(...DynamicTypeSize.accessibility2)
+            .id(toast.id)
+            .padding(.horizontal, 16)
+            .padding(.vertical, 14)
+            .foregroundColor(Asset.Colors.textReversed.swiftUIColor)
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .background(Asset.Colors.backgroundAlert.swiftUIColor)
+            .clipShape(RoundedRectangle(cornerRadius: 8))
+            .shadow(color: .black.opacity(0.1), radius: 8, x: 0, y: 2)
+            .shadow(color: .black.opacity(0.05), radius: 4, x: 0, y: 4)
+            .accessibilityElement(children: .combine)
+            .padding(.horizontal, 16)
+            .task(id: toast.id) {
+                do {
+                    try await Task.sleep(nanoseconds: 3 * NSEC_PER_SEC)
+                    withAnimation {
+                        self.toast = nil
```

Comment:
this is not really possible to test, right?

Reply:
Right, I don't think there's a way we can trigger this with view inspector.